extras/include_checker.py

```diff
@@ -145,6 +145,7 @@ def print_includes(includes):
 
 def process_source(path, f, all_header, print_suggestion):
     if f in excludes_files:
+        print("Please do not change the includes in '" + f + "'.")
```

Comment:
Please re-phrase: 
```python
print("Not checking file " + f +" as it is in the exclude list. Please do not change the order of includes.")
```

Also, please add a comment to line 27 explaining why files are in the exclude list. For a possible sentence on this, see https://github.com/nest/nest-simulator/pull/280#discussion_r60908734